src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    
+    @Override
+    public void  doWork() {
```

Comment:
There is an extra space between `void` and `doWork`. It should be `public void doWork()`.